.github/workflows/publish-ruby.yaml

```diff
@@ -1,114 +1,61 @@
-name: Build Ruby
+name: Publish Ruby
```

Comment:
Since we're renaming these, maybe we should make them consistent across the different technologies.

Reply:
Good suggestion, patched the .NET one so they also match their containing file